metrics_v2/handler/main.py

```diff
@@ -102,10 +101,8 @@ def process_proto_message(
     message_id: Unique message ID to match jobs to metrics in BQ.
```

Comment:
nit: add a "Returns" section to explain what should be in that tuple and why the second part of the tuple can be empty

Reply:
Done.